src/Core/Test/PathUtilsTests.cs

```diff
@@ -16,36 +16,44 @@
 using FluentAssertions;
 using Microsoft.Python.Core.IO;
 using Microsoft.VisualStudio.TestTools.UnitTesting;
+using NSubstitute;
 
 namespace Microsoft.Python.Core.Tests {
     [TestClass]
     public class PathUtilsTests {
```

Comment:
All tests in this class fail on my system.

Reply:
```
  Message: 
    Expected zipPath to be 
    "\path\foo\baz\test.zip" with a length of 22, but 
    "" has a length of 0, differs near "" (index 0).
```

etc

Reply:
Yes, sorry changed a line and didn't commit. Also removed the UNC test because I felt it was redundant.

Reply:
Still fails, though with:

```
    Expected relativeZipPath to be 
    "test\a.py", but 
    "test/a.py" differs near "/a." (index 4).
```

If these paths need to be `/` separated because they're zip-internal, then you can't use `Path.Combine` on them. Better to just write them out by hand than passing them to `Combine`.